task-sdk/pyproject.toml

```diff
@@ -213,3 +213,6 @@ pythonpath = "tests"
 # Keep temporary directories (created by `tmp_path`) for 2 recent runs only failed tests.
 tmp_path_retention_count = "2"
 tmp_path_retention_policy = "failed"
+
+[project.optional-dependencies]
+docs = ["apache-airflow-devel-common[docs]"]
```

Comment:
Nit: this should be around L 146 so it's with the other deps

Additionally this should be a `dependency-groups`, as it doesn't ever make sense for a user to do `pip install "apache-airflow-task-sdk[docs]"` (which is what optional deps allows)

Reply:
changed it.